src/ui/toolbar/MainToolBar.cc

```diff
@@ -68,8 +70,12 @@ MainToolBar::MainToolBar()
     }
     // Get screen DPI to manage font sizes on different platforms
     QScreen *srn = QGuiApplication::screens().at(0); // TODO: Find current monitor as opposed to picking first one
-    _dotsPerInch = (qreal)srn->logicalDotsPerInch(); // Font point sizes are based on Mac 72dpi
-
+    if(srn && srn->logicalDotsPerInch() > 50.0) {
+        _dotsPerInch = (qreal)srn->logicalDotsPerInch(); // Font point sizes are based on Mac 72dpi
+    } else {
+        qWarning() << "System not reporting logical DPI.";
```

Comment:
It is if they launch it from the command line. Let me put it this way, can you give me a good reason not to give more detail here? For developers, who see this message, it'd be nice to debug. And I do watch the console to see what QGC's doing, so I appreciate more detail. I don't see what the harm is to add another 5 words here to describe what it did since it couldn't find a logical DPI like: "System not reporting logical DPI. Defaulting to 120ppi." Is that really that bad?